code/game/say.dm

```diff
@@ -62,21 +62,12 @@ GLOBAL_LIST_INIT(freqtospan, list(
 		if(face_name)
 			namepart = "[H.get_face_name()]" //So "fake" speaking like in hallucinations does not give the speaker away if disguised
 		if(!radio_freq)
-			if(H.wear_id)
-				var/datum/job/wearer_job = SSjob.GetJob(H.wear_id.GetJobName())
-				var/wearer_say_span
-				if(wearer_job)
-					wearer_say_span = wearer_job.say_span
+			if(H.wear_id?.GetID())
+				var/obj/item/card/id/idcard = H.wear_id
+				if(idcard.hud_state == "unknown")
```

Comment:
This is defined somewhere (in jobs most likely)

Reply:
no, they're newly added from my recent PR. These are not in jobs.

Reply:
ah, right. my mistake.